### ISSUE TITLE
Runner JSON logs

### ISSUE THREAD

**Issue by otherguy** [robusta-dev/robusta#1048]
We use Kibana for logs and use structured JSON logs where possible.

It would be great if the `runner` could be configured to emit JSON logs.

**Comment by Sheeproid** [comment]
H, Tomer from the Robusta team here, we looked into it and have a proposed solution. Writing it down here so it is visible for contributors.

1. Json logging should be done via the **https://github.com/madzak/python-json-logger** library
2. Json logging should be optional. The user should opt-in for json logging by setting a variable to his generated_values.yaml file: **json_logging: true**

**Comment by otherguy** [comment]
@Sheeproid sounds good to me!

**Comment by itsCheithanya** [comment]
@Sheeproid I would love work on this issue

**Comment by aantn** [comment]
@pavangudiwada did you already create a POC for this?

**Comment by pavangudiwada** [comment]
@aantn Yep, here's the code https://github.com/robusta-dev/robusta/commit/3e35fb1c44fe200b450b726bd970b8cc6b7215dc. Just need some tweaking and testing.

**Comment by itsCheithanya** [comment]
@pavangudiwada Could you guide me on what needs to be done on my end so that I may begin?

**Comment by pavangudiwada** [comment]
@itsCheithanya I might get to this in the next week or sooner. Happy to help if you have other issues that interest you.

**Comment by itsCheithanya** [comment]
I would love to work on any of good first issues

**Comment by otherguy** [comment]
@pavangudiwada any progress on that?
